CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Optimizely Ruby SDK Changelog
 
+## 3.3.2-RC
+December 4th, 2019
+
+### Bug Fixes:
+- BatchEventProcessor will hang on poll after flushing with no event if no event count is greater than 3.
```

Comment:
Please use the commit message for the fixes and reference the PRs here

Reply:
If you are releasing from master Please make sure to reference the other commits since the last release as well